### ISSUE TITLE
UI sounds for 1.4 release

### ISSUE THREAD

**Issue by markgammon** [phetsims/graphing-lines#137]
As part of the work done for https://github.com/phetsims/vegas/issues/77, UI sounds got turned on for this simulation.  Before the sim is published again from master, we should decide whether to turn the UI sounds off, leave them on and design and implement sounds for the rest of the interactions, or publish with a partial sound design.

If the decision is to turn them off, this can be accomplished by adding the following line to the end of the sim's `-main.js` file:

```js
  soundManager.setOutputLevelForCategory( 'user-interface', 0 );
```

@pixelzoom EDIT: It would be preferrable to turn sound off by removing `"supportsSound": true` from package.json.

**Comment by marlitas** [comment]
@pixelzoom will be working on the region and culture publication of the graphing lines sims. @amanda-phet can you let us know what you would like to do in terms of sound for these sims?

**Comment by pixelzoom** [comment]
@amanda-phet To make a decision here, you can test drive on phettest.  The interactive objects that do not have associated sounds are the manipulators (shaded spheres on the graph) and the point tools.   Buttons, checkboxes, and pickers all have default (common code) sounds.

**Comment by pixelzoom** [comment]
Also note that turning off `'user-interface'` category sounds will _not_ turn off vegas (game) sounds.  Specifically, these sounds in the Line Game screen will still be enabled:  correct or incorrect answer, level completed, reward for perfect score.

**Comment by pixelzoom** [comment]
Unless PhET has time to do a sound design for the manipulators and point tools, this is a trade-off between 2 suboptimal user experiences.  The manipulators and point tools are the main things that the user interacts with, so it's unfortunate that they have no sound.  But the vegas sounds are essential for a game, and it would seem strange to have only games sounds, while still having a sound button in the navigation bar, and an Audio tab in Preferences. So my recommendation is to publish "as is", with the a partial sound design for the 1.4 release.  And address sound for the manipulators and point tools in a future release.

**Comment by pixelzoom** [comment]
@amanda-phet pointed out that there are grab-and-release (aka drag-and-drop) sounds used in many sims.  The tambo (common code) sound files are:

```
import grab_mp3 from '../../../tambo/sounds/grab_mp3.js';
import release_mp3 from '../../../tambo/sounds/release_mp3.js';
```

I pointed out that some sims (my-solar-system, keplers-laws, balloons,... ) do not use those sounds, and (for whatever reason) have their own sounds.   For example, in solar-system-common:

```ts
import Grab_Sound_mp3 from '../../sounds/Grab_Sound_mp3.js';
import Release_Sound_mp3 from '../../sounds/Release_Sound_mp3.js';
```

Because there is no support for this pattern in scenery's DragListener and KeyboardDragListener, all sims with grab-and-release sounds also duplicate some variation of this code (whether they use tambo sounds or not):

```ts
    const dragClipOptions = {
      initialOutputLevel: .... // a value between 0 and 1
    };
    const grabClip = new SoundClip( Grab_Sound_mp3, dragClipOptions );
    const releaseClip = new SoundClip( Release_Sound_mp3, dragClipOptions );
    soundManager.addSoundGenerator( grabClip );
    soundManager.addSoundGenerator( releaseClip );
...
    const start = () => grabClip.play();
    const end = () => releaseClip.play();
```

Setting for initialOutputLevel are also all over the place in sims.  I found everything from 0.2 to 1.0.  It seems like this is also something that needs to be standardized (with option to override).

So... We decided to use the tambo sounds in graphing-lines, and create yet-another duplicate piece of code to wire those sounds up the DragListeners for point tools and manipulators.  For levels, I chose `initialOutputLevel: 0.4` because it was used in sound-waves and wave-interference, and seemed reasonable when I tested graphing-lines.   This work was completed in the above commits, and is available for testing in main or [1.4.0-dev.5](https://phet-dev.colorado.edu/html/graphing-lines/1.4.0-dev.5/phet/graphing-lines_all_phet.html).

@arouinfar over to you for review.
